src/ucs/datastruct/mpool.c

```diff
@@ -265,11 +291,17 @@ void ucs_mpool_grow(ucs_mpool_t *mp, unsigned num_elems)
 void *ucs_mpool_get_grow(ucs_mpool_t *mp)
 {
     ucs_mpool_data_t *data = mp->data;
+    unsigned num_elems;
 
     ucs_mpool_grow(mp, data->elems_per_chunk);
     if (mp->freelist == NULL) {
         return NULL;
     }
+    ucs_assert(data->chunks != NULL);
+    num_elems             = ucs_min(data->elems_per_chunk,
+                                    data->chunks->num_elems);
+    /* Calculate num of elems for next growing */
```

Comment:
move this comment to before `ucs_assert(data->chunks != NULL);`

Reply:
fixed